libfwupd/fwupd-release.c

```diff
@@ -770,7 +770,7 @@ fwupd_release_get_trust_flags (FwupdRelease *release)
 {
 	FwupdReleasePrivate *priv = GET_PRIVATE (release);
 	g_return_val_if_fail (FWUPD_IS_RELEASE (release), 0);
-	return priv->trust_flags;
+	return priv->flags & 0x3;
```

Comment:
I think I'd rather see this as `#define` to `0x3` put near where the `TRUST_METADATA` and `TRUST_PAYLOAD` are in case other trust flags are added later so this doesn't get forgotten.

Reply:
Done, agreed.